packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -334,10 +334,12 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
         const oldPositionEdge = oldPositions[key];
         const newPositionEdge = newPositions[key];
 
-        if (oldPositionEdge && newPositionEdge) {
+        if (oldPositionEdge !== undefined && newPositionEdge !== undefined) {
           if (oldPositionEdge.toFixed(2) !== newPositionEdge.toFixed(2)) {
```

Comment:
Change this to === and then have it return true, then in the outer section where it returns true, have that return false. I think that makes sense. So it would look like this instead.

'''  private _comparePositions(oldPositions: ICalloutPositon, newPositions: ICalloutPositon) {
    for (const key in newPositions) {
      // This needs to be checked here and below because there is a linting error if for in does not immediately have an if statement
      if (newPositions.hasOwnProperty(key)) {
        const oldPositionEdge = oldPositions[key];
        const newPositionEdge = newPositions[key];

        if (oldPositionEdge !== undefined && newPositionEdge !== undefined) {
          if (oldPositionEdge.toFixed(2) === newPositionEdge.toFixed(2)) {
            return true;
          }
        } 
      }
    }
    return false;
  }
```

Reply:
@joschect That could cause it to return true on the first match and exit the `for` loop, before it finished comparing all properties, wouldn't it?

Reply:
Yes, good catch, just ignore me!